test/fixtures/output/connect/plugin/install/remote-bad-md5.golden

```diff
@@ -0,0 +1,10 @@
+Beginning installation.
+
+Using the only available Confluent Platform installation at "test/fixtures/input/connect/confluent-7.3.3".
+Do you want to install this plugin into "test/fixtures/input/connect/confluent-7.3.3/share/confluent-hub-components"? (y/n): 
+License:
+Example License 1.0
+url-to-license-goes-here
+I agree to this software license agreement. (y/n): 
+Installing Integration Test Plugin 0.1.0, provided by Confluent, Inc.
+Error: md5 checksum for downloaded archive (7a920f4187137251171abcd7c7dd9844) does not match checksum in manifest (12345) for plugin "integration-test-plugin"
```

Comment:
```suggestion
Error: MD5 checksum for downloaded archive (7a920f4187137251171abcd7c7dd9844) does not match checksum in manifest for plugin "integration-test-plugin" (12345)
```

Reply:
I made these uppercase, but it feels awkward to me to relocate the second checksum to the end, since both checksums are supposed to be for the plugin's archive.